raaghu-components/src/rds-comp-database-connection/rds-comp-database-connection.tsx

```
@@ -131,6 +131,7 @@ const RdsCompDatabaseConnection = (props: RdsCompDatabaseConnectionProps) => {
                               <div className="form-group">
                                  <RdsTextArea
                                     label="Database URL"
```

ESLint

[error] Expected indentation of 48 spaces but found 36.

(indent)